v03_pipeline/lib/tasks/write_imported_callset.py

```diff
@@ -62,50 +66,64 @@ def requires(self) -> list[luigi.Task]:
         ]
 
     def create_table(self) -> hl.MatrixTable:
-        mt = import_callset(
-            self.callset_path,
-            self.reference_genome,
-            self.dataset_type,
-        )
-        filters_path = None
-        if (
-            Env.EXPECT_WES_FILTERS
-            and not self.skip_expect_filters
-            and self.dataset_type.expect_filters(
-                self.sample_type,
+        if self.clone(WriteValidationErrorsForRunTask).complete():
```

Comment:
The control flow is a little wonky... this task isn't really a dependency because it cannot run before we either import or validate the callset, but we do want to avoid re-running (we have retries enabled in airflow) if it it exists.

Reply:
doesn't this imply that theres something wrong with our overall control flow? Its a bit unclear to me what the actual issue is, but it feels like if we split our some steps into their own tasks that would fix this issue more explicitly

Reply:
I think, in some sense, yes.  This block is effectively preventing _airflow_ from triggering a retry due to a validation error.  We probably don't want that to be on the luigi side and should try to put it into `airflow`.... I think with potentially an additional task that checks for this file first.

Reply:
I will remove the block from these tasks